tests/tests/test-class-alerts.php

```diff
@@ -382,22 +382,15 @@ function test_save_new_alert_no_nonce() {
 		wp_set_current_user( $user_id );
 
 		$alerts = new Alerts( $this->plugin );
-		try {
-			$_POST['wp_stream_trigger_author'] = 'me';
-			$_POST['wp_stream_trigger_context'] = 'posts-post';
-			$_POST['wp_stream_trigger_action'] = 'edit';
-			$_POST['wp_stream_alert_type'] = 'highlight';
 
-			$this->expectException( 'WPAjaxDieStopException' );
-			$this->_handleAjax( 'save_new_alert' );
-		} catch ( \WPAjaxDieContinueException $e ) {
-			$exception = $e;
-			// Check that the exception was thrown.
-			$this->assertTrue( isset( $exception ) );
+		$_POST['wp_stream_trigger_author'] = 'me';
```

Comment:
I simplified some of these tests as they did a weird combination of expecting an exception but also intercepting it before it is shown to PHPUnit.